localization/l10n/bundle.l10n.json

```diff
@@ -99,9 +99,6 @@
   "Move Up": "Move Up",
   "Move Down": "Move Down",
   "Delete": "Delete",
-  "Error loading designer": "Error loading designer",
```

Comment:
Automatically got removed because they're duplicates (see lines 162, 164, and 168)